components/glyphicons/glyphicons-en.html

```diff
@@ -0,0 +1,818 @@
+---
+title: Glyphicons
+description: Documentation and working examples for Glyphicons
+language: en
+altLangPage: glyphicons-fr.html
+dateModified: 2023-07-06
+layout: no-container
+---
+<div class="container">
+	<h1>{{ page.title }}</h1>
+
+	<span class="wb-prettify all-pre hide"></span>
+
+	<!--<h2>Purpose</h2>
+	<p>This adds Bootstrap Glyphicons to a Canada.ca page. Bootstrap includes 260 glyphs from the Glyphicon Halflings set. Glyphicons Halflings are normally not available for free, but their creator has made them available for Bootstrap free of cost. As a thank you, you should include a link back to <a href="https://glyphicons.com/">Glyphicons</a> whenever possible.</p>-->
+
+	<h2>Working examples</h2>
+
+	<div class="row wb-eqht">
+		<div class="col-lg-2 col-sm-3 col-xs-6 well">
+			<p><span class="glyphicon glyphicon-asterisk" aria-hidden="true"></span><br>glyphicon<br>glyphicon-asterisk</p>
```

Comment:
For 1.1.1 conformance, we would need to name each icon. We can find those name in the glyphicons upstream documentation here: https://glyphicons.com/sets/basic/

Also, we would need to add `<code>` around the CSS class used to implement the feature

```suggestion
			<p>Asterisk<br><span class="glyphicon glyphicon-asterisk" aria-hidden="true"></span><br><code>glyphicon glyphicon-asterisk</code></p>
```